pyproject.toml

```
@@ -5,7 +5,7 @@ description = "Corretores do Einstein Floripa (Simulinho, SIMUFSC, SIMUENEM, Fee
 authors = ["Vale do Silicio <vale@einsteinfloripa.com.br>"]
 license = "MIT"
 readme = "README.md"
-packages = [{include = "corretor_simulados"}]
+packages = [{include = "src"}]
```

@matos1396 Cara, a gente poderia mudar talvez para SRC?

```suggestion
packages = [{include = "SRC"}]
```

---

@vianaz resolvido

---

@vianaz resolvido